src/community/wfs-freemarker/src/main/java/org/geoserver/wfsfreemarker/html/HtmlGetFeatureOutputFormat.java

```diff
@@ -0,0 +1,68 @@
+/* (c) 2024 Open Source Geospatial Foundation - all rights reserved
+ * This code is licensed under the GPL 2.0 license, available at the root
+ * application directory.
+ */
+package org.geoserver.wfsfreemarker.html;
+
+import static org.geoserver.wms.featureinfo.FreeMarkerTemplateManager.OutputFormat.HTML;
+
+import java.io.IOException;
+import java.io.OutputStream;
+import java.util.List;
+import org.geoserver.config.GeoServer;
+import org.geoserver.platform.GeoServerResourceLoader;
+import org.geoserver.platform.Operation;
+import org.geoserver.platform.ServiceException;
+import org.geoserver.wfs.WFSGetFeatureOutputFormat;
+import org.geoserver.wfs.request.FeatureCollectionResponse;
+import org.geoserver.wms.WMS;
+import org.geoserver.wms.featureinfo.FreeMarkerTemplateManager;
+import org.geoserver.wms.featureinfo.HTMLTemplateManager;
+import org.geotools.feature.FeatureCollection;
+
+/**
+ * A GetFeature response handler specialized in producing HTML data for a GetFeature request through
+ * the {@link FreeMarkerTemplateManager}.
+ */
+public class HtmlGetFeatureOutputFormat extends WFSGetFeatureOutputFormat {
```

Comment:
```suggestion
public class HTMLGetFeatureOutputFormat extends WFSGetFeatureOutputFormat {
```

In java acronyms should stay all uppercase, although this practice is not always respected, in GeoServer HTML is consistently capitalized, e.g.:

```
> find . -name "*html*.java"
> find . -name "*HTML*.java"
./rest/src/main/java/org/geoserver/rest/converters/FreemarkerHTMLMessageConverter.java
./extension/mapml/src/main/java/org/geoserver/mapml/MapMLHTMLOutputFormat.java
./extension/mapml/src/main/java/org/geoserver/mapml/MapMLHTMLResponse.java
./extension/mapml/src/main/java/org/geoserver/mapml/MapMLHTMLMap.java
./extension/importer/rest/src/main/java/org/geoserver/importer/rest/converters/ImportTaskHTMLMessageConverter.java
./extension/importer/rest/src/main/java/org/geoserver/importer/rest/converters/ImportContextHTMLMessageConverter.java
./extension/importer/rest/src/main/java/org/geoserver/importer/rest/converters/ImportDataHTMLMessageConverter.java
./web/core/src/main/java/org/geoserver/web/LoginFormHTMLInclude.java
./web/core/src/test/java/org/geoserver/web/LoginFormHTMLIncludeTest.java
./wms/src/main/java/org/geoserver/wms/featureinfo/HTMLFeatureInfoOutputFormat.java
./wms/src/test/java/org/geoserver/wms/featureinfo/HTMLFeatureInfoOutputFormatTest.java
./community/imagemap/src/main/java/org/vfny/geoserver/wms/responses/map/htmlimagemap/HTMLImageMapResponse.java
./community/imagemap/src/main/java/org/vfny/geoserver/wms/responses/map/htmlimagemap/EncodeHTMLImageMap.java
./community/imagemap/src/main/java/org/vfny/geoserver/wms/responses/map/htmlimagemap/HTMLImageMapWriter.java
./community/imagemap/src/main/java/org/vfny/geoserver/wms/responses/map/htmlimagemap/HTMLImageMapMapProducer.java
./community/imagemap/src/test/java/org/vfny/geoserver/wms/responses/map/htmlimagemap/HTMLImageMapTest.java
./community/ogcapi/ogcapi-features/src/main/java/org/geoserver/ogcapi/v1/features/GetFeatureHTMLMessageConverter.java
./community/ogcapi/dggs/ogcapi-dggs/src/main/java/org/geoserver/ogcapi/v1/dggs/DGGSFeatureHTMLMessageConverter.java
./community/ogcapi/ogcapi-maps/src/main/java/org/geoserver/ogcapi/v1/maps/HTMLMap.java
./community/ogcapi/ogcapi-maps/src/main/java/org/geoserver/ogcapi/v1/maps/HTMLMapMessageConverter.java
./community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/AnnotatedHTMLMessageConverter.java
./community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/HTMLResponseBody.java
./community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/HTMLExtensionCallback.java
./community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/AbstractServiceHTMLMessageConverter.java
./community/ogcapi/ogcapi-core/src/main/java/org/geoserver/ogcapi/AbstractHTMLMessageConverter.java
./community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/AbstractItemsHTMLMessageConverter.java
./community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/SearchHTMLMessageConverter.java
./community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/ItemHTMLMessageConverter.java
./community/oseo/oseo-stac/src/main/java/org/geoserver/ogcapi/v1/stac/ItemsHTMLMessageConverter.java
./community/features-templating/features-templating-ows/src/main/java/org/geoserver/featurestemplating/ows/wfs/HTMLTemplateResponse.java
./community/features-templating/features-templating-ows/src/main/java/org/geoserver/featurestemplating/ows/wms/HTMLTemplateFeatureInfo.java
./community/features-templating/features-templating-ows/src/test/java/org/geoserver/featurestemplating/response/HTMLGetComplexFeatureResponseTest.java
./community/features-templating/features-templating-ows/src/test/java/org/geoserver/featurestemplating/response/HTMLGetSimpleFeaturesResponseWFSTest.java
./community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/writers/XHTMLTemplateWriter.java
./community/features-templating/features-templating-core/src/main/java/org/geoserver/featurestemplating/readers/XHTMLTemplateReader.java
./community/features-templating/features-templating-ogcapi/src/test/java/org/geoserver/featurestemplating/response/HTMLComplexFeaturesResponseApiTest.java
./community/features-templating/features-templating-ogcapi/src/test/java/org/geoserver/featurestemplating/response/HTMLSimpleFeaturesResponseAPITest.java
```